src/util/prometheus.js

```diff
@@ -508,6 +508,18 @@ class Prometheus {
         type: 'gauge',
         labelNames: ['destType', 'feature'],
       },
+      {
+        name: 'braze_identify_calls_count_without_filter',
```

Comment:
same for this metric, using a counter instead of a gauge would be more appropriate